guide/private-key-management/single-user-schemes.md

```diff
@@ -0,0 +1,335 @@
+---
+layout: guide
+title: Personal schemes
+description: An overview of personal private key management schemes.
+nav_order: 2
+parent: Private key management
+permalink: /guide/private-key-management/single-user-schemes/
+main_classes: -no-top-padding
+image: /assets/images/guide/private-key-management/schemes/page-personal-schemes.jpg
+---
+
+<!--
+
+Editor's notes
+
+Descriptions of schemes suitable for a single user.
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/private-key-management/personal.jpg"
+   retina = "/assets/images/guide/private-key-management/personal@2x.jpg"
+   mobile = "/assets/images/guide/private-key-management/personal-mobile.jpg"
+   mobileRetina = "/assets/images/guide/private-key-management/personal-mobile@2x.jpg"
+   alt-text = "Personal schemes header illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Personal schemes
+
+Now let's dive in and look at the various schemes that might be suitable for a personal wallet application with a single user to understand the differences at a high level.
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/schemes/personal-schemes.jpg"
+   retina = "/assets/images/guide/private-key-management/schemes/personal-schemes@2x.jpg"
+   alt-text = ""
+   caption = "Cloud backup, manual backup , signing-device, threshold-signatures and multi-key."
+   width = 2012
+   height = 400
+%}
+
+- [Automatic cloud backup]({{ '/guide/private-key-management/single-user-schemes/#automatic-cloud-backup' | relative_url }})
+- [Manual backup / Recovery phrase]({{ '/guide/private-key-management/single-user-schemes/#recovery-phrase' | relative_url }})
+- [External signing device]({{ '/guide/private-key-management/single-user-schemes/#external-signing-device' | relative_url }})
+- [Threshold signatures / Key-sharing]({{ '/guide/private-key-management/single-user-schemes/#threshold-signatures--key-sharing' | relative_url }})
+- [Multi-key]({{ '/guide/private-key-management/single-user-schemes/#multi-key' | relative_url }})
+
+***
+
+## Automatic cloud backup
```

Comment:
Do you think it's helpful what is meant by cloud backup? To me it always sounds like some online storage, but in reality it's the keychain (or password manager).